rackcity/views/decommission_views.py

```diff
@@ -287,3 +290,27 @@ def decommissioned_asset_many(request):
         {"assets": serializer.data},
         status=HTTPStatus.OK,
     )
+
+
+@api_view(['POST'])
+@permission_classes([IsAuthenticated])
+def decommissioned_asset_page_count(request):
+    """
+    Return total number of pages according to page size, which must be
+    specified as query parameter.
+    """
+    change_plan = None
+    if request.query_params.get("change_plan"):
+        (change_plan, response) = get_change_plan(
+            request.query_params.get("change_plan")
+        )
+        if response:
+            return response
+    if change_plan:
+        return get_page_count_response_for_cp(request, change_plan)
```

Comment:
I think I copied this from before decommissioning was added to change plan - @clairecfu do you have SA?